src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -605,6 +610,10 @@ namespace OpenMS
     PeakType::IntensityType calculateTIC() const;
 
 protected:
+
+    /// Peak1d Tuple
+    Container spectra();
```

Comment:
it's peaks. You are in the spectrum class. A spectrum does not contain multiple spectra

Reply:
Sure I'll correct that.